.vagrant/machines/default/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:739b7777-ca61-410b-90a0-1872b60c87bf
```

Comment:
@olivermel please remove all .vagrant files and add ".vagrant" to the .gitignore file